Kitodo/src/main/java/de/sub/goobi/forms/AktuelleSchritteForm.java

```diff
@@ -338,7 +341,8 @@ public String uploadFromHomeAlle() throws DataException, IOException {
         List<URI> fertigListe = this.myDav.uploadAllFromHome(doneDirectoryName);
         List<URI> geprueft = new ArrayList<>();
         /*
-         * die hochgeladenen Prozess-IDs durchlaufen und auf abgeschlossen setzen
+         * die hochgeladenen Prozess-IDs durchlaufen und auf abgeschlossen
```

Comment:
If you change a comment - translate it to English.

Reply:
This is just the code formatter reformatting the comment, I didn't change it.